modules/util/src/main/native/android/c/util.h

```diff
@@ -41,4 +41,4 @@ JavaVM* getGraalVM();
 
 jclass substrateGetUtilClass();
 
-jboolean debugAttach;
```

Comment:
still need to include a guard in this file, e.g. #ifndef ATTACH_UTIL

Reply:
Done